tests/drivers/watchdog/wdt_basic_reset_none/Kconfig

```diff
@@ -0,0 +1,16 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
+
+config TEST_WDT_MAX_WINDOW_TIME
+	int "Watchdog Max Value"
+	default 1000
+	help
+	  Set the watchdog max window time to feed the watchdog.
+
+config TEST_WDT_SLEEP_TIME
+	int "Watchdog Sleep Time"
+	default 500
+	help
+	  Set the test sleep time between watchdog feeds.
```

Comment:
What are the units for these times?  It would be nice to include here in the help description